cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/NetworkClientImpl.kt

```diff
@@ -183,12 +182,19 @@ private class NetworkClientImpl(
             .get()
             .build()
 
-        val response = httpClient.newCall(request).execute()
+//        val response = httpClient.newCall(request).execute()
+        val response = Response.Builder()
+            .code(444)
+            .protocol(Protocol.HTTP_1_1)
+            .request(Request.Builder().url(url).build())
+            .message("")
+            .body(ResponseBody.create("application/json".toMediaTypeOrNull(), ""))
+            .build()
```

Comment:
Why are we building the response with a code `444` ?

Reply:
because this is the change for a manual test, I told you about it